src/main/java/org/jfrog/hudson/pipeline/scripted/steps/CollectEnvStep.java

```diff
@@ -37,7 +40,9 @@ public Execution(CollectEnvStep step, StepContext context) throws IOException, I
 
         @Override
         protected Boolean run() throws Exception {
-            step.getEnv().collectVariables(env, build, listener);
+            CollectEnvExecutor collectEnvExecutor = new CollectEnvExecutor(build, listener, ws,
+                    step.getEnv(), env);
```

Comment:
Can be inlined and still be short:
```suggestion
            CollectEnvExecutor collectEnvExecutor = new CollectEnvExecutor(build, listener, ws, step.getEnv(), env);
```